packages/client/types/constEnums.ts

```diff
@@ -129,7 +129,7 @@ export const enum Layout {
   LAYOUT_GRID = 8, // 1x
   ROW_GUTTER = 16, // 2x
   SETTINGS_MAX_WIDTH = 768,
-  TASK_COLUMNS_MAX_WIDTH = 1304 // (4 x 296 card max-width) + (5 x 24 - gutters around cols)
+  TASK_COLUMNS_MAX_WIDTH = 1360
```

Comment:
curious, what's the reason for the change? what's the calculation?

Reply:
When the width was 1304, I found the Done column did not fit:

![](https://user-images.githubusercontent.com/1879975/94175634-ccc9c900-fe4b-11ea-8d21-71e55b58cd9d.png)

The number 1360 was calculated as follows:

1.  Card width: 296
2.  Left and right padding of 12px defined on [DraggaleTask](https://github.com/ParabolInc/parabol/blob/master/packages/client/containers/TaskCard/DraggableTask.tsx#L13)
3.  Somehow the column width has 15px addition which I couldn't find reference.

Therefore (296 + 12 + 12 + 15) \* 4 = 1340 for 4 columns. Finally 10px left/right padding defined on [TaskColumns](https://github.com/ParabolInc/parabol/blob/master/packages/client/components/TaskColumns/TaskColumns.tsx#L26). Hence 1360.